ecommerce/OrderController.php

```diff
@@ -156,6 +164,71 @@ public function get_order_by_id() {
 		);
 	}
 
+	/**
+	 * Mark an order as paid.
+	 *
+	 * This function verifies a nonce for security, constructs a payload object with
+	 * the order ID, note, and payment status, and updates the payment status of the order
+	 * to 'paid'. It sends a JSON response indicating the success or failure of the operation.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @return void
+	 */
+	public function order_mark_as_paid() {
+		if ( ! tutor_utils()->is_nonce_verified() ) {
+			$this->json_response( tutor_utils()->error_message( 'nonce' ), null, HttpHelper::STATUS_BAD_REQUEST );
+		}
+
+		if ( ! current_user_can( 'manage_options' ) ) {
+			$this->json_response( tutor_utils()->error_message( HttpHelper::STATUS_UNAUTHORIZED ), null, HttpHelper::STATUS_UNAUTHORIZED );
+		}
+
+		$params = array(
+			'order_id' => Input::post( 'order_id' ),
+			'note'     => Input::post( 'note' ),
+		);
+
+		// Validate request.
+		$validation = $this->validate( $params );
+		if ( ! $validation->success ) {
+			$this->json_response(
+				tutor_utils()->error_message( HttpHelper::STATUS_BAD_REQUEST ),
+				$validation->errors,
+				HttpHelper::STATUS_BAD_REQUEST
+			);
+		}
+
+		do_action( 'tutor_before_order_mark_as_paid', $params );
+
+		$payload                 = new \stdClass();
+		$payload->order_id       = $params['order_id'];
+		$payload->note           = $params['note'];
+		$payload->payment_status = $this->model::PAYMENT_PAID;
+
+		if ( empty( $payload->order_id ) ) {
+			$this->json_response(
+				__( 'Order Id is required', 'tutor' ),
```

Comment:
`__( 'Order Id is required', 'tutor' ),` I think it would be ID or id?

Reply:
im using ID then